src/transformState.js

```diff
@@ -5,7 +5,30 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const action of actions) {
+    switch (true) {
```

Comment:
don't use `true` in switch case, 
```suggestion
    switch (action.type) {
```